ci-operator/config/openshift/deadmanssnitch-operator/openshift-deadmanssnitch-operator-master.yaml

```diff
@@ -1,30 +1,47 @@
-base_images:
-  base:
-    name: "4.4"
-    namespace: ocp
-    tag: base
-  openshift_release_golang-1.10:
-    name: release
-    namespace: openshift
-    tag: golang-1.10
-  openshift_release_golang-1.13:
-    name: release
-    namespace: openshift
-    tag: golang-1.13
 build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.10
+  from_repository: true
 images:
 - dockerfile_path: build/Dockerfile
-  from: base
-  to: deadmanssnitch-operator
+  to: unused
 resources:
   '*':
+    limits:
+      memory: 4Gi
     requests:
       cpu: 100m
       memory: 200Mi
+tests:
+- as: coverage
+  commands: |
+    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
+    make coverage
+  container:
+    from: src
```

Comment:
Do you mind explaining what `src` is here? I never figured it out.

Reply:
Yeah, it's confusing.

- `src` is created automatically by the CI pipeline. It's `root` plus a clone of the source repository, sitting on a merge commit from the proposed PR.
- `root` comes from `build_root`, which we're [defining](https://github.com/openshift/release/blob/cb98a2465a9519b5132ad2a13b0ba417eb6c2029/ci-operator/config/openshift/deadmanssnitch-operator/openshift-deadmanssnitch-operator-master.yaml#L6-L7) such that it gets the config from [the source repository's `.ci-operator.yaml`](https://github.com/openshift/deadmanssnitch-operator/blob/f9cfccd45c4f99b9768c2df8d44c96f2457b169b/.ci-operator.yaml).

Clear as mud?